src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -87,10 +86,12 @@ export class CourseUpdateComponent implements OnInit {
         private router: Router,
         private featureToggleService: FeatureToggleService,
         private accountService: AccountService,
-        private fb: FormBuilder,
     ) {}
 
     ngOnInit() {
+        this.profileService.getProfileInfo().subscribe((profileInfo) => {
+            this.ltiEnabled = profileInfo.activeProfiles.includes(PROFILE_LOCALVC);
+        });
```

Comment:
Why don't you add this below? We already call the profile service in this onInit.

Reply:
just overlooked this, feel free to move it down